README.md

```diff
@@ -320,6 +321,11 @@ See also [A comparison of operating systems written in Rust](https://github.com/
 
 * [hEngine](https://github.com/hashintel/hash/tree/main/packages/engine) - A Rust-implemented computational simulation engine, supporting large-scale agent-based modelling, with simulation logic written in JavaScript and Python.
 
+### Social network
```

Comment:
This should be "Social networks" not "Social network" in line with the plural form for everything else.